### ISSUE TITLE
GNU licence in dependencies 

### ISSUE THREAD

**Issue by pettzilla1** [pdfminer/pdfminer.six#739]
The original PDF miner only had a BSD 2 licence required in the dependencies , six requires chardet which reduces the usability of the package due to it being GNU might I suggest its replaced with  "charset_normalizer" which is MIT , while also being well maintained.

**Comment by pietermarsman** [comment]
I'm not an expert in licences. Can you rephrase your post to explain better what the situation is and what you want?

**Comment by pettzilla1** [comment]
Though pdfminer.six has an MIT license using it as part of a service causes problems as one of the dependencies in this case “chardet” has a gnu license which means you would have to distribute your source code with whatever system you make which isn’t ideal so technically pdfminer.six is made to have this requirement as you need to install “chardet” for it to work.

"charset_normalizer" Is MIT and has a similar functionality to “chardet” by using it instead I don’t believe there would then be a requirement to distribute source code with any system that used pdfminer.six.

would it be possible to use "charset_normalizer" instead of “chardet” ? it would make the overall package a lot more useable.

**Comment by pietermarsman** [comment]
Seems like it is [an improvement on many levels](https://pypi.org/project/charset-normalizer/): more coverage, faster and better licence. 

Also, the [requests library is using it](https://github.com/psf/requests/blob/main/setup.py#L61) as well so it is probably mature enough. 

Will create a PR for this.

**Comment by pietermarsman** [comment]
@pettzilla1 can you review my PR?

**Comment by pettzilla1** [comment]
I've reviewed @pietermarsman  but don't have write access, So I believe someone else will need to as well

**Comment by pettzilla1** [comment]
@jstockwin would you be able to review the PR?

**Comment by jstockwin** [comment]
Done :+1:

**Comment by pettzilla1** [comment]
@pietermarsman thanks for the help with this issue ,  The PR has been reviewed 👍
